src/Components/Alert/Components/Filters/Size.tsx

```diff
@@ -0,0 +1,231 @@
+      customSizeRanges.width != DEFAULT_RANGE ||
+      customSizeRanges.height != DEFAULT_RANGE
+    ) {
+      updateCriteriaAttribute("sizes", [])
+    }
+  }
+
+  return (
+    <>
+      <Join separator={<Spacer y={1} />}>
+        <Text variant="sm-display">Size</Text>
+
+        <RadioGroup
+          defaultValue={metric}
+          onSelect={handleSelectMetric}
+          flexDirection="row"
+          my={2}
+        >
+          <Radio value="cm" label="cm" flex={1} />
+          <Radio value="in" label="in" flex={1} />
+        </RadioGroup>
+
+        <Flex flexDirection="column">
+          {predefinedSizes.map(({ name, displayName }, index) => {
+            return (
+              <Checkbox
+                key={index}
+                onSelect={selected => toggleSizeSelection(selected, name)}
+                selected={sizes.includes(name)}
+                my={1}
+              >
+                {displayName}
+              </Checkbox>
+            )
+          })}
+        </Flex>
+
+        <Clickable
+          mt={1}
+          textDecoration="underline"
+          textAlign="left"
+          onClick={() => setShowCustom(prevShowCustom => !prevShowCustom)}
+        >
+          <Text variant="xs">{showCustom ? "Hide" : "Show"} custom size</Text>
+        </Clickable>
+
+        {showCustom && (
+          <>
+            <Text mt={1}>Width</Text>
+            <Flex alignItems="flex-end">
+              <NumericInput
+                label={metricLabel}
+                name="width_min"
+                min="0"
+                step="1"
+                value={getRangeValue(customSize.width[0])}
+                onChange={handleInputChange("width", 0)}
+              />
+              <Spacer x={1} />
+              <NumericInput
+                label={metricLabel}
+                name="width_max"
+                min="0"
+                step="1"
+                value={getRangeValue(customSize.width[1])}
+                onChange={handleInputChange("width", 1)}
+              />
+            </Flex>
+
+            <Text mt={1}>Height</Text>
+            <Flex alignItems="flex-end">
+              <NumericInput
+                label={metricLabel}
+                name="height_min"
+                min="0"
+                step="1"
+                value={getRangeValue(customSize.height[0])}
+                onChange={handleInputChange("height", 0)}
+              />
+
+              <Spacer x={1} />
+
+              <NumericInput
+                label={metricLabel}
+                name="height_max"
+                min="0"
+                step="1"
+                value={getRangeValue(customSize.height[1])}
+                onChange={handleInputChange("height", 1)}
+              />
+            </Flex>
+
+            <Button
+              mt={1}
+              variant="primaryGray"
+              onClick={handleSetSizesClick}
+              width="100%"
+            >
+              Set size
+            </Button>
```

Comment:
Do we have a design for this part? The button looks quite big 🤔  


Could we simply remove the button and set the custom height and width directly when the user types in the input?

![Screenshot 2023-11-23 at 11 24 11](https://github.com/artsy/force/assets/4691889/45919403-cbf6-43da-8bf6-35b538584ce3)

Reply:
Same as above, pinged Barney to take a look!

Reply:
We've agreed to make this button smaller for desktop and keep it 100% for mobile @olerichter00:

<img width="686" alt="image" src="https://github.com/artsy/force/assets/3934579/cbbb30e8-a9ee-4ac8-b249-321811b41f9e">